### ISSUE TITLE
Error(s) blocking validation

### ISSUE THREAD

**Issue by dunglex** [validator/validator#2055]
### Summary

I use w3 validator to validate my page.
Here is the page url: https://career.homecredit.vn/en/
The validator url: https://validator.w3.org/check?uri=https%3A%2F%2Fcareer.homecredit.vn%2Fen%2F&charset=%28detect+automatically%29&doctype=Inline&group=0
The error message is: Sorry! This document cannot be checked.
Error: I got the following unexpected response when trying to retrieve <https://career.homecredit.vn/en/>:

500 Can't connect to career.homecredit.vn:443 (certificate verify failed)

If you made recent changes to your domain name (DNS) configuration, you may also want to [check that your domain records are correct](http://www.squish.net/dnscheck/), or ask your hosting company to do so.

I believe this is wrong error because I can access the page without any ssl error.

### URL for document that reproduces the problem

https://validator.w3.org/check?uri=https%3A%2F%2Fcareer.homecredit.vn%2Fen%2F&charset=%28detect+automatically%29&doctype=Inline&group=0

### HTML/CSS/SVG snippet that reproduces the problem

```html
This is error when validating a page by URI
```

### Checker error messages

```shell
Sorry! This document cannot be checked.
Error
I got the following unexpected response when trying to retrieve <https://career.homecredit.vn/en/>:

500 Can't connect to career.homecredit.vn:443 (certificate verify failed)

If you made recent changes to your domain name (DNS) configuration, you may also want to check that your domain records are correct, or ask your hosting company to do so.
```

### Steps to reproduce

1. Open https://validator.w3.org/ 
2. Paste https://career.homecredit.vn/en/ into Address
3. Click Check button

### Expected behavior

Should validate the uri

### Actual behavior

Return error 500 Can't connect to career.homecredit.vn:443 (certificate verify failed)

**Comment by sideshowbarker** [comment]
Yeah there’s nothing we can do to from the checker side to fix that problem. The https://career.homecredit.vn server is misconfigured.

**Comment by sideshowbarker** [comment]
Actually, instead try https://validator.w3.org/nu/?doc=https://career.homecredit.vn/. It’ll work.

**Comment by dunglex** [comment]
This url works: https://validator.w3.org/nu/?doc=https%3A%2F%2Fcareer.homecredit.vn%2Fvn%2F
This url doesn't work: https://validator.w3.org/check?uri=https%3A%2F%2Fcareer.homecredit.vn%2F
This isn't domain misconfigured.

**Comment by sideshowbarker** [comment]
> This url works: [validator.w3.org/nu?doc=https%3A%2F%2Fcareer.homecredit.vn%2Fvn%2F](https://validator.w3.org/nu/?doc=https%3A%2F%2Fcareer.homecredit.vn%2Fvn%2F) This url doesn't work: [validator.w3.org/check?uri=https%3A%2F%2Fcareer.homecredit.vn%2F](https://validator.w3.org/check?uri=https%3A%2F%2Fcareer.homecredit.vn%2F) This isn't domain misconfigured.

Yeah, you’re right
